web/src/app/components/modal/modal.component.ts

```diff
@@ -279,11 +281,16 @@ export class ModalComponent extends BaseModal implements OnInit {
       } else {
         this.showProgress();
         this.modalSvc.submitForm(this.defendantAndCaseForm.value)
-        .subscribe((data: any) =>  {
-          // wait for data success and then close modal
-          this.modalService.destroy()
-          this.closeModal()
-        });
+        .subscribe((data: any) => {
+            // wait for data success and then close modal
+            this.modalService.destroy()
+            this.closeModal();
+          },
+          (err) => {
+            console.error('Error: ' + JSON.stringify(err));
+            this.modalService.destroy()
+            this.closeModal();
+          });
```

Comment:
Now that the submit button goes to the service we need to handle the error and it can't connect